tests/molecules/H2_FDLR_opt/H2.info.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
This is an output file, it should not be checked-in (unless output is included for reference)